package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cheapreats/ts-sdk",
-  "version": "1.12.0",
+  "version": "1.12.5",
```

Comment:
why is it .5 and not .1 ?

Reply:
Whenever I npm install the version seems to increment by .5, I'll change for .1 then 👍